content/en/integrations/guide/collect-sql-server-custom-metrics.md

```diff
@@ -181,6 +181,11 @@ You can also specify:
 
 **Note**: The `proc_only_if` guard condition is useful for HA scenarios where a database can move between servers.
 
+If your custom metrics are not appearing in Datadog, check the Agent log file. If you see the following error: `Could not call procedure <PROCEDURE_NAME>: You must supply -1 parameters for this stored procedure`, it could be one of the following issues:
```

Comment:
```suggestion
### Troubleshooting

If your custom metrics are not appearing in Datadog, check the Agent log file. If you see the following error: `Could not call procedure <PROCEDURE_NAME>: You must supply -1 parameters for this stored procedure`, it could be one of the following issues:
```